Dockerfile

```diff
@@ -10,16 +10,31 @@ RUN apk add python3 py3-pip git \
 CMD git ls-files --cached -z -- '*.rst' \
     | xargs -0 -- python3 -m sphinxlint --enable all --disable trailing-whitespace --max-line-length 2000
 
-FROM php:${PHP_VERSION}-cli-alpine${ALPINE_VERSION} as php
+FROM php:${PHP_VERSION}-cli-alpine${ALPINE_VERSION} as base
```

Comment:
we use different extensions, `dom/mbstring/json/tokenizer`. shall we explicitly mention them somewhere?

Reply:
I believe all required extensions are installed by default, because I've been using Docker locally regularly in our project and it worked. If I remember correctly (currently AFK) I have remote Docker interpreter configured in IDE. And release images are built with the same setup.

But maybe it's a good idea to run `composer check-platform-reqs` somewhere in the process 🤔. Then we only should list extensions in Composer's `require` section.

But as far as I see only JSON and tokenizer extensions are required and I believe these are built-in.

Reply:
json + tokenizer are build-in, but dom+mbstring beneficial (XML progress report, faster execution of mb_ methods (that we do not need to polyfill)

> Then we only should list extensions in Composer's require section.

no, the suggested too, pls. unblocks feature + performance

Reply:
@keradus everything is installed (json, tokenizer, mbstring, dom), so I think we don't need do anything. If we need it explicitly, then I can make improve it later, for now it's non-blocking.

```shell
$ d run --rm -it -v $(pwd):/var/www --entrypoint="" ghcr.io/elbformat/php-cs-fixer:1.2.3-php8.3 sh

/var/www # php -m
[PHP Modules]
Core
ctype
curl
date
dom
fileinfo
filter
ftp
hash
iconv
json
libxml
mbstring
mysqlnd
openssl
pcre
PDO
pdo_sqlite
Phar
posix
random
readline
Reflection
session
SimpleXML
sodium
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
zlib

[Zend Modules]

/var/www # php -r 'echo mb_strtolower("Ü");'
ü
```